src/Symfony/Component/HttpKernel/DataCollector/EventDataCollector.php

```diff
@@ -27,9 +27,9 @@
  */
 class EventDataCollector extends DataCollector implements LateDataCollectorInterface
 {
-    protected $dispatcher;
-    private $requestStack;
-    private $currentRequest;
+    private EventDispatcherInterface $dispatcher;
```

Comment:
this can be null according to the constructor

Reply:
Fixed.